docs/eprintf.3

```diff
@@ -0,0 +1,64 @@
+'\" t
+.\"     Title: eprintf
+.\"    Author: [see the "AUTHOR(S)" section]
+.\" Generator: Asciidoctor 1.5.7.1
+.\"      Date: 2017-09-16
+.\"    Manual: CS50 Programmer's Manual
+.\"    Source: CS50
+.\"  Language: English
+.\"
+.TH "EPRINTF" "3" "2017-09-16" "CS50" "CS50 Programmer\(aqs Manual"
+.ie \n(.g .ds Aq \(aq
+.el       .ds Aq '
+.ss \n[.ss] 0
+.nh
+.ad l
+.de URL
+\fI\\$2\fP <\\$1>\\$3
+..
+.als MTO URL
+.if \n[.g] \{\
+.  mso www.tmac
+.  am URL
+.    ad l
+.  .
+.  am MTO
+.    ad l
+.  .
+.  LINKSTYLE blue R < >
+.\}
+.SH "NAME"
+eprintf \- prints an error message to stderr
+.SH "SYNOPSIS"
+.sp
+\fB#include <cs50.h>\fP
+.sp
+\fBvoid eprintf(const string format, ...);\fP
+.SH "DESCRIPTION"
+.sp
+Prints an error message formatted like \fBprintf(3)\fP to standard error, prefixing it with file and line number from which the function was called. See \fBprintf(3)\fP for more on formatting.
+.SH "EXAMPLE"
+.sp
+.if n .RS 4
+.nf
+int main(void)
+{
+    eprintf("This line of code should be executed\(rsn");
+
+    int i;
+    for (i = 0; i < 50; i++)
+    {
+        printf("i is %d\(rsn", i);
+    }
+
+    eprintf("Expected i to be 49 but it is actually %d\(rsn", i);
+}
+.fi
+.if n .RE
+.SH "SEE ALSO"
+.sp
+.if n .RS 4
+.nf
+printf(3)
+.fi
+.if n .RE
```

Comment:
Are we sure we want to maintain these in troff format?

Reply:
It's not like we change them that frequently and it'd be nice not to require `ruby` and `asciidoctor` to be installed especially for people who are building and installing from source?